drivers/ieee802154/ieee802154_cc13xx_cc26xx_subg.c

```diff
@@ -697,8 +708,8 @@ static struct ieee802154_cc13xx_cc26xx_subg_data
 		.rxConf = {
 			.bAutoFlushIgnored = true,
 			.bAutoFlushCrcErr = true,
-			.bIncludeHdr = true,
 			.bAppendRssi = true,
+			.bAppendStatus = true,
```

Comment:
The status field is fairly usesless. See "Table 25-180. Receive Status Byte Bit Field" in the TRM.